Removed stacktrace from error message

MarekB01 opened the pull request:
Removed stacktrace from error message
Also made it user friendly

https://github.com/BroadleafCommerce/QA/issues/4690

MarekB01 committed:
BroadleafCommerce/QA#4690

Removed stacktrace from error message

oleksiimiroshnyk commented on admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/web/handler/AdminMappingExceptionResolver.java:
```diff
@@ -49,8 +49,9 @@ public ModelAndView resolveException(HttpServletRequest request, HttpServletResp
-                StringBuilder sb2 = new StringBuilder();
-                appendStackTrace(ex, sb2);
+                StringBuilder sb2 = new StringBuilder("An error was occured");
+                //uncomment next line to show stacktrace when error is occured
```
maybe instead of commenting out add some property to show or not to show stacktrace ?

MarekB01 committed:
BroadleafCommerce/QA#4690

Removed stacktrace from error message
Made a property to enable/disable stacktrace in error message
